src/inc/clrconfigvalues.h

```diff
@@ -648,15 +648,21 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_HillClimbing_GainExponent,
 ///
 #ifdef FEATURE_TIERED_COMPILATION
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_TieredCompilation, W("TieredCompilation"), 1, "Enables tiered compilation")
+RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_TC_StartupTier_QuickJit, W("TC_StartupTier_QuickJit"), 0, "For methods that would be jitted, enable using quick JIT in the startup tier.")
```

Comment:
Nit: Should this match the naming convention used for the rest of the switches in this group?

Reply:
I changed them to `TC_` because some of the names were getting too long. There is a 64 char limit including the `COMPlus_` prefix (which could be increased as well). I figured for the more advanced config options a `TC_` prefix would be ok.

Reply:
ok

Reply:
I would be nice to rename all options to have the `TC_` prefix for consistency before it is too late.

Reply:
Oh ok, I was doing that as part of the other PR, I'll move those here

Reply:
Fixed in latest commit